doc/09-object-types.md

```diff
@@ -1510,6 +1510,7 @@ Configuration Attributes:
   Name            |Description
   ----------------|----------------
   severity        |**Optional.** The minimum severity for this log. Can be "debug", "notice", "information", "warning" or "critical". Defaults to "warning".
+  facility        |**Optional.** Defines the facility to use for syslog entries. This can either be a facility name like "LOG_LOCAL0" or a corresponding number like "128". Defaults to "LOG_USER".
```

Comment:
I wonder if we should also add constants for all well-known syslog facilities (i.e. all the predefined LOG_* values). Something like Syslog.USER, Syslog.User, SyslogFacilityUser, FacilityUser. I don't really know about how to name them just yet.

In either case we should also document all valid LOG_* values.

Reply:
I would also like to switch to LOG_DAEMON as default because this facility is meant for "system daemons without separate facility value".
For naming I am fine with LOG_*, if you want to create constants using the most simple naming schema would be great as something like SyslogFacilityUser is more complicated than LOG_USER.

Reply:
I would keep the default which was hardcoded ever since. Now users can freely define it on their own.